docs/index.md

```diff
@@ -7,11 +7,11 @@
 * [Installation](INSTALLATION.md) - the steps to install the Helm charts
 * [Operation](OPERATION.md) - how to upgrade applications, scale your cluster, and update resources
 * [Platforms support](PLATFORMS.md) - how to allow support for different platforms
-* [Examples](examples/)
+* Examples
   * [How to deploy an EFK stack to Kubernetes](examples/logging/efk/EFK.md)
   * [Implementation of an NFS Server for Bitbucket](examples/storage/nfs/NFS.md)
   * [Local storage - utilizing AWS EBS-backed volumes](examples/storage/aws/LOCAL_STORAGE.md)
   * [Shared storage - utilizing AWS EFS-backed filesystem](examples/storage/aws/SHARED_STORAGE.md)
 
 *** 
-* Go to [README.md](../README.md)
+* Go to [README.md](https://github.com/atlassian-labs/data-center-helm-charts/blob/master/README.md)
```

Comment:
If they are viewing the file on the `https://atlassian-labs.github.io/data-center-helm-charts/` domain this will send them back to the repository view.

Is there a way how to convince the links to work in both places (source view & github pages)?

Reply:
As discussed IRL let's focus on getting the regular documentation accessible from the repository done first.
The `/docs/index.md` can just link the the `README.md` file for now.